examples/scripts/project/_common/fork_project.py

```diff
@@ -0,0 +1,159 @@
+""" Example script for creating a copy of an existing project with a new name """
+
+import os
+from pathlib import Path
+from typing import Dict, Any
+import argparse
+from argparse import Namespace
+import tamr_toolbox as tbox
+from tamr_toolbox import utils as tbu
+from tamr_toolbox.utils.client import Client
+from tamr_toolbox.utils.operation import Operation
+
+
+def export_from_tamr(client: Client, *, project_name: str, export_path: str,) -> Operation:
+    """
+    This function sets path for project artifacts export from Tamr and makes the call to execute
+    the export action.
+    Export path defaults to "project-movement/<project_name>" if no user-defined value is passed.py
+
+    Args:
+        client: an instance of Tamr unify client object
+        project_name: name of the project to be exported from Tamr
+        export_path: export path - must be accessible to the VM hosting Tamr 
+
+    Returns:
+        operation for project export api call
+    """
+    project = client.projects.by_name(project_name)
+    if not export_path:
+        export_path = os.path.join(
+            Path(__file__).resolve().parent, f"project-movement/{project_name}"
+        )
+    return tbox.project.export_artifacts(project=project, artifact_directory_path=export_path)
+
+
+def main(
+    *,
+    project_name: str,
+    new_name: str,
+    postfix: str,
+    new_ud_name: str,
+    export_path: Path,
+    overwrite: bool,
+    instance_connection_info: Dict[str, Any],
+):
+    """
+    This function creates a fork copy of a Tamr project by exporting the target project 
+    and importing it back to Tamr under a new name
+
+    Example scripts for creating a copy of an existing project with the new project name:
+        - as "_copy"-postfixed name of the target project:
+            python fork_project.py --export_path /path/to/export-file-dir/ --project_name <target_project_name> --postfix _copy
+        - specified explicitly:
+            python fork_project.py --export_path /path/to/export-file-dir/ --project_name <target_project_name> --new_name <new_project_name>
```

Comment:
Can you move these examples to the docstring at the top of the file rather than in the docstring of the main function? Something like
```
"""Example script for creating a copy of an existing project with a new name

To copy a project as "_copy"-postfixed name of the target project:
    python fork_project.py --export_path /path/to/export-file-dir/ --project_name <target_project_name> --postfix _copy
To specify new project name explicitly:
    python fork_project.py --export_path /path/to/export-file-dir/ --project_name <target_project_name> --new_name <new_project_name>
"""
```

Reply:
I was trying to push a new commit but I get this error message:
```
git push origin TBOX226-fork_project
To https://github.com/Datatamer/tamr-toolbox.git
 ! [rejected]        TBOX226-fork_project -> TBOX226-fork_project (non-fast-forward)
error: failed to push some refs to 'https://github.com/Datatamer/tamr-toolbox.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```
Any hints as to how I can resolve this?

Reply:
I had to do a rebase so I understand I will have to do a pull and then push with a `-f` flag to push new commits.

Reply:
This looks like your remote branch is ahead of your local branch, i.e. there's at least one commit there that you don't have locally.  I see the last commit on your remote is "made requested changes by skalish and ycharm to fork_project.py", do you see that on your local?  Try using the `git log --graph --oneline` command to see your latest commits.